lib/transaction/payload/commitmenttxpayload.js

```diff
@@ -84,6 +87,10 @@ CommitmentTxPayload.fromBuffer = function fromBuffer(rawPayload) {
     .read(constants.SHA256_HASH_SIZE)
     .toString('hex');
 
+  if (payload.version >= constants.HASH_QUORUM_INDEX_REQUIRED_VERSION) {
```

Comment:
@shumkov I think you're checking the wrong version here. The qcTx version didn't change, only the qfcommit message version contained inside it.
```suggestion
  if (payload.qfcVersion >= constants.HASH_QUORUM_INDEX_REQUIRED_VERSION) {
```